tests/testthat/test-shinytest2-tm_a_regression.R

```diff
@@ -0,0 +1,69 @@
+testthat::test_that("e2e - tm_a_regerssion: ", {
+  skip_if_too_deep(5)
+
+  # needed untill this is merged https://github.com/insightsengineering/teal/pull/1198
+  require(shinytest2)
+
+  app <- app_driver_tm_a_regression()
+
+  app$expect_no_shiny_error()
+  # app$open_url()
+
+  testthat::expect_equal(
+    app$get_text("#teal-main_ui-root-active_tab > li.active > a"),
+    "Regression"
+  )
+
+  # Check MAIN encoding panel
+  # DO WE NEED any function for checking top of the encoding panel part?
+  encoding_dataset <- app$get_text("#teal-main_ui-root-regression > div > div.col-md-3 > div.well > div > span")
+  testthat::expect_match(encoding_dataset, "Dataset", fixed = TRUE)
+  testthat::expect_match(encoding_dataset, "CO2", fixed = TRUE)
```

Comment:
What do you think about making this a bit more broad?

```suggestion
  encoding_dataset <- app$get_text("#teal-main_ui-root-regression .help-block")
  testthat::expect_match(encoding_dataset, "Dataset:[\n ]*CO2", all = FALSE)
```

Reply:
really nice, thanks